test/nexus_test.go

```diff
@@ -312,6 +313,11 @@ func TestSyncOperationFromWorkflow(t *testing.T) {
 	tc := newTestContext(t, ctx)
 
 	op := temporalnexus.NewSyncOperation("op", func(ctx context.Context, c client.Client, outcome string, o nexus.StartOperationOptions) (string, error) {
+		require.NotPanicsf(t, func() {
```

Comment:
Would rather check the metrics labels and logger tags but this is fine for now. Can open an issue for later.